airflow/api_fastapi/app.py

```diff
@@ -70,6 +75,25 @@ def init_views(app) -> None:
     app.include_router(ui_router)
     app.include_router(public_router)
 
+    dev_mode = os.environ.get("DEV_MODE", False) == "true"
+
+    directory = "./airflow/ui/dev" if dev_mode else "./airflow/ui/dist"
+
+    templates = Jinja2Templates(directory=directory)
+
+    app.mount(
+        "/static",
+        StaticFiles(
+            directory=directory,
+            html=True,
+        ),
+        name="webapp_static_folder",
+    )
+
+    @app.get("/webapp/{rest_of_path:path}", response_class=HTMLResponse, include_in_schema=False)
```

Comment:
One point for fast api that handles `rest_of_path`. No need to keep in sync the front-end router and the backend url.